cart.css

```diff
@@ -0,0 +1,47 @@
+.cart-card-container {
+    padding: 1rem;
+    height: 70vh;
+    display: flex;
```

Comment:
your cart content may increase than the 70vh, you may use min height

Reply:
got it, will do it rn.